spec/services/record_declaration_spec.rb

```diff
@@ -345,6 +345,15 @@
         end
       end
 
+      context "has_passed is invalid text" do
+        let(:has_passed) { "no_supported" }
+
+        it "returns error" do
+          expect(service).to be_invalid
+          expect(service.errors.messages_for(:has_passed)).to eq(["The property '#/has_passed' must be true or false"])
```

Comment:
Are we worried about case here? In the ticket it says `TRUE` / `FALSE`.

Reply:
Additionally if we change this spec group to `type: :model` we can check the validations with:

```
expect(service).to allow_value("true", "false").for(:has_passed).with_message("The property...")
```

which might be a bit more concise.

Reply:
@peteryates 
Added fix for capitalisation.

Reply:
Speaking with @cwrw, I've changed back to only accepting lowercase to stay consistent with the rest of the API. We'll update the guidance to reflect this.